Source/CombatExtended/CombatExtended/Projectiles/BulletCE.cs

```diff
@@ -21,7 +21,9 @@ private void LogImpact(Thing hitThing, out BattleLogEntry_RangedImpact logEntry)
 					hitThing,
 					intendedTarget,
 					equipmentDef,
-					def);
+					def
+                    //CoverDef Missing!
```

Comment:
CE doesn't use cover in the sense vanilla does

Reply:
What do you suggest to pass as the argument?
The method needs a variable of the type CoverDef.

Reply:
Should be safe to null

Reply:
Is null.